azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -17,6 +17,7 @@
 package azkaban.security;
 
 import static azkaban.Constants.ConfigurationKeys.AZKABAN_SERVER_NATIVE_LIB_FOLDER;
+import static org.apache.hadoop.hive.metastore.api.hive_metastoreConstants.*;
```

Comment:
We have just 1 constant that we are using from `hive_metastoreConstants` right? Should we just import that?

Reply:
Yes, it seems that we only have META_TABLE_STORAGE under that should be imported. But intellij auto-imported it as "org.apache.hadoop.hive.metastore.api.hive_metastoreConstants.*". Should I change that and start a new pull request?

Reply:
If you follow the developer setup instruction in the README, import the google style xml and use the latest Intellj, it appears that it will import individual classes. Could you give it a try?

Reply:
Yes that solves the problem. Already updated the code